libraries/ui/src/main/java/androidx/media3/ui/PlayerControlView.java

```diff
@@ -1574,14 +1574,25 @@ private void seekToTimeBarPosition(Player player, long positionMs) {
   }
 
   private void onFullScreenButtonClicked(View v) {
-    if (onFullScreenModeChangedListener == null) {
+    updateIsFullScreen(!isFullScreen);
+  }
+
+  /**
+   * Updates whether the controller is in full screen to its icon
+   * and reports it to to the listener.
+   * @param isFullScreen If the view is in full screen.
```

Comment:
I would reword this javadoc (see PlayerView comment)

Reply:
I updated this javadoc too:
Updates whether the controller is in fullscreen, changing its fullscreen icon and reports it to to the listener.

For isFullscreen equals true the icon will be set to ``exo_styled_controls_fullscreen_exit`` or else ``exo_styled_controls_fullscreen_enter``.